src/docs/reference/modules/OpenId/README.md

```diff
@@ -49,9 +49,72 @@ Available settings are:
 - Enable Authorization Endpoint.
 - Enable Logout Endpoint.
 - Enable User Info Endpoint.
-- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3>
+- Allow Password Flow: It requires that the Token Endpoint is enabled. More info at <https://tools.ietf.org/html/rfc6749#section-1.3.3> 
+
+Typescript request example using axios:
+
+``` typescript 
+    
+  const data: any =
+  {
+        username: username,
+        password: password,
+        // must read from config
+        client_id: 'client_id',
+        client_secret: 'client_secret',
```

Comment:
I know axios can be used in both browser and server apps, but I think most people will try to use this snippet in a browser app. In this case, we shouldn't encourage them to use a `client_secret` here, just in case.

Reply:
I added a note message as a reminder